server/api/api.raml

```diff
@@ -78,12 +78,11 @@ types:
       low-space-ratio?: number
       high-space-ratio?: number
       scheduler-max-waiting-operator?: integer
-      disable-raft-learner?: boolean

```

Comment:
We don't need it ?

Reply:
`disable-raft-learner` is deprecated in previous PR I think.

Reply:
yes.